index.html

```diff
@@ -352,38 +351,155 @@ <h3 class="contributor__subtitle">
+                                    </div>
+    
+                                    <div class="maintainers__more-info">
+                                        <div class="maintainers__stats">
+                                            <div class="maintainers__stats-item">
+                                                <img class="maintainers__stats-icon" src="/assets/svg/commit.svg" />
+                                                <span class="maintainers__stats-text">141</span>
+                                            </div>
+                                        </div>
     
+                                        <div class="maintainers__contacts">
+                                            <div class="maintainers__contacts-item maintainers__contacts-item--name">
+                                                <a class="maintainers__contacts-link" href="https://github.com/fballiano" target="_blank">fballiano</a>
+                                            </div>
+                                            <div class="maintainers__contacts-item maintainers__contacts-item--location">
+                                                <span class="maintainers__contacts-text">Arrecife, Spain</span>
+                                            </div>
+                                            <div class="maintainers__contacts-item maintainers__contacts-item--link">
+                                                <a class="maintainers__contacts-link maintainers__contacts-link--limit" href="https://fabrizioballiano.com" target="_blank">https://fabrizioballiano.com</a>
+                                            </div>
+                                        </div>
+                                    </div>
+                                </div>
+                                
                                 <div class="maintainers__person">
                                     <div class="maintainers__view">
-                                        <img class="maintainers__image" src="/assets/images/maintainers/daniel.png" />
-                                        <div class="maintainers__name">
-                                            Daniel Fahlke
+                                        <img class="maintainers__image" src="/assets/images/maintainers/lee.png" />
+                                        <div class="maintainers__name">Lee Saferite</div>
+                                    </div>
+    
+                                    <div class="maintainers__more-info">
+                                        <div class="maintainers__stats">
+                                            <div class="maintainers__stats-item">
+                                                <img class="maintainers__stats-icon" src="/assets/svg/commit.svg" />
+                                                <span class="maintainers__stats-text">75</span>
```

Comment:
why did it go down from 117?

Reply:
that's what https://github.com/OpenMage/magento-lts/graphs/contributors says, i don't know

Reply:
reverted to 117